Drivers/Src/MotorController.c

```diff
@@ -11,13 +11,21 @@
 #define MASK_CC_ERR 1<<2 //checks velocity on 2 bit
 #define MASK_OVER_SPEED_ERR 1<<24 //check if motor overshot max RPM on 24 bit
 
+/**
+ * BSP_CAN_INIT requires a tx and rx event handler, and this driver doesn't actually require any tx/rx event handling. As such, this 
+ * is a dummy function just to satisfy the BSP layer's constraints.
```

Comment:
This works, but a more elegant solution might be to modify the BSP to not do anything for the rx or tx event handlers if you pass in `NULL`